### PR TITLE
Fetch cube

### PR DESCRIPTION
**Pull request by giacomociti**
renamed fetch-constraint and check-constraint to fetch-cube and check-cube

fetch-cube includes also the cbd of cube

### CODE REVIEW

**Comment by giacomociti** [comment]
I'm thinking that maybe `fetch-metadata` and `check-metadata` would be better names, highlighting that a cube is made of observations (data) and metadata (including the constraint)

File: .changeset/unlucky-plants-carry.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"barnard59-cube": major
```
Comment (tpluscode) [inline .changeset/unlucky-plants-carry.md]: I know it's technically a breaking change but no has had the chance to use this yet :)

```suggestion
"barnard59-cube": patch
```

File: packages/cube/README.md [hunk 2]
```diff
@@ -18,15 +18,15 @@ TBD
 
 ## Cube validation
 
-The following pipelines retrieve and validate cube observations and their constraints.
+The following pipelines retrieve and validate cube observations and their metadata.
```
Comment (tpluscode) [inline packages/cube/README.md]:
```suggestion
The following pipelines retrieve and validate cube observations and metadata.
```

File: packages/cube/README.md [hunk 3]
```diff
@@ -35,15 +35,15 @@ npx barnard59 cube fetch-constraint \
...
-Pipeline `check-constraint` validates the input constraint against the shapes provided with the `profile` variable (the default profile is https://cube.link/latest/shape/standalone-constraint-constraint).
+Pipeline `check-metadata` validates the input metadata against the shapes provided with the `profile` variable (the default profile is https://cube.link/latest/shape/standalone-constraint-constraint).
 
-The pipeline reads the constraint from `stdin`, allowing input from a local file (as in the following example) as well as from the output of the `fetch-constraint` pipeline (in most cases it's useful to have the constraint in a local file because it's needed also for the `check-observations` pipeline).
+The pipeline reads the metadata from `stdin`, allowing input from a local file (as in the following example) as well as from the output of the `fetch-metadata` pipeline (in most cases it's useful to have the metadata in a local file because it's needed also for the `check-observations` pipeline).
 
 ```bash
-cat myConstraint.ttl \
-| npx barnard59 cube check-constraint \
+cat metadata.ttl \
```
Comment (tpluscode) [inline packages/cube/README.md]:
```suggestion
cat cube.ttl \
```

File: packages/cube/manifest.ttl [hunk 4]
```diff
@@ -21,18 +21,18 @@
...
 
 
-<command/cube/fetch-constraint>
+<command/cube/fetch-metadata>
   a b59:CliCommand ;
-  b59:command "fetch-constraint" ;
-  rdfs:label "Retrieves cube constraint from SPARQL endpoint" ;
-  b59:source "barnard59-cube/pipeline/fetch-constraint.ttl" ;
+  b59:command "fetch-metadata" ;
+  rdfs:label "Retrieves cube and constraint from SPARQL endpoint" ;
```
Comment (tpluscode) [inline packages/cube/manifest.ttl]:
```suggestion
  rdfs:label "Retrieves cube and its constraint shape from SPARQL endpoint" ;
```

File: packages/cube/manifest.ttl [hunk 5]
```diff
@@ -21,18 +21,18 @@
...
 .
 
-<command/cube/check-constraint>
+<command/cube/check-metadata>
   a b59:CliCommand ;
-  b59:command "check-constraint" ;
-  rdfs:label "Validate input constraint against profile shapes" ;
-  b59:source "barnard59-cube/pipeline/check-constraint.ttl" ;
+  b59:command "check-metadata" ;
+  rdfs:label "Validate input cube and constraint against profile shapes" ;
```
Comment (tpluscode) [inline packages/cube/manifest.ttl]:
```suggestion
  rdfs:label "Validate input cube and its constraint against profile shapes" ;
```

**Review by tpluscode — CHANGES_REQUESTED** [review]

**Review by tpluscode — APPROVED** [review]

**Pull request merged**